libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -639,13 +648,15 @@ def validate_environment(cls, values: Dict) -> Dict:
             client_options=values.get("client_options"),
             transport=transport,
         )
-        values["async_client"] = genaix.build_generative_async_service(
-            credentials=values.get("credentials"),
-            api_key=google_api_key,
-            client_info=client_info,
-            client_options=values.get("client_options"),
-            transport=transport,
-        )
+
+        if _is_event_loop_running():
+            values["async_client"] = genaix.build_generative_async_service(
+                credentials=values.get("credentials"),
+                api_key=google_api_key,
+                client_info=client_info,
+                client_options=values.get("client_options"),
+                transport=transport,
+            )
```

Comment:
maybe good to add an `else` case setting `values["async_client"] = values.get("async_client", None)` just to be explicit (pydantic does funny things sometimes)